### PR TITLE
Update laboratory1.sql

### PR DESCRIPTION
**Pull request by dbisstudent**

### CODE REVIEW

File: test/Tereshchenko_Igor/laboratory1.sql [hunk 1]
```diff
@@ -1,2 +1,151 @@
 -- LABORATORY WORK 1
 -- BY Tereshchenko_Igor
```
Comment (igortereshchenko) [inline test/Tereshchenko_Igor/laboratory1.sql]: Delete these comments

File: test/Tereshchenko_Igor/laboratory1.sql [hunk 2]
```diff
@@ -1,2 +1,151 @@
...
+
+SELECT                      -- final projection, we don't need CUST_ID
+  TRIM(PROD_NAME),
+  TRIM(CUST_NAME)
+FROM (
+        SELECT              -- select all combination of customer and product
+          CUSTOMERS.CUST_ID,
+          CUST_NAME,
+          PROD_NAME
+        FROM CUSTOMERS, PRODUCTS
```
Comment (igortereshchenko) [inline test/Tereshchenko_Igor/laboratory1.sql]: Full 

- [x] fields name

- [ ] distinct

**Review by igortereshchenko — CHANGES_REQUESTED** [review]

File: test/Tereshchenko_Igor/laboratory1.sql (hunk 1 above)
Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql]: Fixed

File: test/Tereshchenko_Igor/laboratory1.sql (hunk 2 above)
Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql]: Fixed

Comment (igortereshchenko) [reply test/Tereshchenko_Igor/laboratory1.sql]: Not all fixed

**Review by igortereshchenko — CHANGES_REQUESTED** [review]

File: test/Tereshchenko_Igor/laboratory1.sql (hunk 2 above)
Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql]: Sorry fixed

Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql]: Fixed

**Review by dbisstudent — COMMENTED** [review]
Fixed

File: test/Tereshchenko_Igor/laboratory1.sql (hunk 2 above)
Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql]: done

**Review by dbisstudent — COMMENTED** [review]
Fixed

File: test/Tereshchenko_Igor/laboratory1.sql (hunk 2 above)
Comment (dbisstudent) [reply test/Tereshchenko_Igor/laboratory1.sql]: dgddgdgd

**Review by dbisstudent — COMMENTED** [review]
gddgzgdg gstgdf

**Review by igortereshchenko — APPROVED** [review]

**Pull request closed without merging**